cli/src/opts/forge.rs

```diff
@@ -25,6 +25,10 @@ use regex::Regex;
 static GH_REPO_REGEX: Lazy<Regex> =
     Lazy::new(|| Regex::new("[A-Za-z\\d-]+/[A-Za-z\\d_.-]+").unwrap());
 
+static GH_REPO_PREFIX_REGEX: Lazy<Regex> = Lazy::new(|| {
+    Regex::new(r"((git@)|(git\+https://)|(https://))?(github|gitlab).com(/|:)?").unwrap()
```

Comment:
Any way we can make this more generic so we don't need to update this a lot? For example, this would be missing BitBucket

Reply:
Good shout, updated to make this regex more generic.